### PR TITLE
Export Reva runtime

### PR DESCRIPTION
**Pull request by labkode**
Move main logic to package.

### CODE REVIEW

**Comment by refs** [comment]
🎉

File: cmd/revad/runtime/runtime.go [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/trace"
+)
+
+// Run runs a reva server with the given config file and pid file.
+func Run(config string, pidFile string) {
```
Comment (butonic) [inline cmd/revad/runtime/runtime.go]: Can we pass in the map[string]interface{} here so we don't have to read the config from disk? An io.Reader would work as well, but passing in a map would be the least overhead.

Comment (labkode) [reply cmd/revad/runtime/runtime.go]: @butonic of course :)

**Comment by labkode** [comment]
@butonic @tboerger We'll add later log redirection so you can consume it natively in OCIS.

**Pull request merged**